web/pages/requests.tsx

```diff
@@ -40,12 +44,13 @@ export const getServerSideProps = async (
   context: GetServerSidePropsContext
 ) => {
   const supabase = createServerSupabaseClient(context);
+  const user = await supabase.auth.getUser();
```

Comment:
nit: unnecessary getUser request. the `session` object below already has the user inside of it. 

`user: session.user,`

Reply:
good catch, thanks!